.github/workflows/weekly-enviroment.yml

```diff
@@ -23,6 +23,8 @@ on:
 env:
   WORKING_DIR: deploy/weekly-environment
   SCRIPTS_DIR: deploy/weekly-environment/scripts/benchmarks/kspm_vanilla
+  ALERTS_DIR: deploy/weekly-environment/scripts/alerts
+  HC_DIR: deploy/weekly-environment/scripts/health_checks
```

Comment:
Not sure about the health_checks directory

Reply:
When the health checks are called?